build.gradle

```diff
@@ -23,7 +23,7 @@ buildscript {
         classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.3'
         classpath 'org.ajoberstar:gradle-git:1.3.2'
         classpath 'com.bmuschko:gradle-vagrant-plugin:2.0'
-        classpath 'com.bmuschko:gradle-docker-plugin:3.0.6'
+        classpath 'com.bmuschko:gradle-docker-plugin:3.0.7'
```

Comment:
This update fixes the false task failure when building a Docker image on new versions of Docker.